### PR TITLE
Note effect of cluster/disabled option.

### PR DESCRIPTION
**Pull request by pillarsdotnet**
The [code](https://github.com/wazuh/wazuh/blob/master/src/config/cluster-config.c#L60) that parses the [cluster/disabled](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/cluster.html#disabled) option sets a [local](https://github.com/wazuh/wazuh/blob/master/src/config/cluster-config.c#L35) variable [`disable_cluster_info = 1;`](https://github.com/wazuh/wazuh/blob/master/src/config/cluster-config.c#L62), whose [effect](https://github.com/wazuh/wazuh/blob/master/src/config/cluster-config.c#L85) is the same as [setting cluster/hidden = yes](https://github.com/wazuh/wazuh/blob/master/src/config/cluster-config.c#L65).

It does not, so far as I can see, prevent the cluster from starting.

### CODE REVIEW

**Comment by mgmacias95** [comment]
Hello @pillarsdotnet,

Thanks for your contribution but the changes you propose aren't correct. Let me explain you why:

* The `disabled` option is handled [here](https://github.com/wazuh/wazuh/blob/588888a097d4c81f904ca292111b34c7f9bb723b/framework/scripts/wazuh-clusterd.py#L247-L248). If it's enabled (value = no) the cluster will start running, otherwise (value = yes) it will stop running.
* The default value of this option can be found [here](https://github.com/wazuh/wazuh/blob/588888a097d4c81f904ca292111b34c7f9bb723b/etc/templates/config/generic/cluster.template#L12).
* The `hidden` option is only used to hide cluster information in wazuh alerts. You can find where this is processed [here](https://github.com/wazuh/wazuh/blob/master/src/analysisd/format/to_json.c#L58-L67).

In conclusion, we can't change the documentation since these two values aren't related to each other.

Best regards,
Marta

**Comment by pillarsdotnet** [comment]
Thanks to @mgmacias95  for pointing out the python parsing code which is separate and distinct from the c-language parsing code.

As for defaults, would the following configuration snippet result in an enabled or disabled cluster?

```xml
<ossec_config>
  <cluster>
    <nodes>
      <node>1.2.3.4</node>
    </nodes>
  </cluster>
</ossec_config>

**Comment by JuanjiJG** [comment]
Hi @pillarsdotnet,

That configuration snippet won't enable the cluster. You need to specify the `<disabled>no</disabled>` tag, along with other such as `<key>`. More information can be found [here](https://documentation.wazuh.com/current/user-manual/manager/wazuh-cluster.html)

Regards,
Juanjo

**Comment by pillarsdotnet** [comment]
I read the documentation, but I question its accuracy.  From the [code](https://github.com/wazuh/wazuh/blob/master/framework/wazuh/configuration.py), it would appear that a missing `<disabled>` tag is the same as setting `<disable>no</disabled>`

**Comment by mgmacias95** [comment]
Hello @pillarsdotnet,

If the `disabled` tag is not present, the cluster won't start. No error is thrown right now but that's a known bug (https://github.com/wazuh/wazuh/issues/1178) we're already working on. By default, the cluster will always be disabled unless otherwise specified.

Best regards,
Marta
